common/app/views/fragments/inlineJSNonBlocking.scala.html

```diff
@@ -30,4 +30,32 @@
 // Ophan pageview ID and browser ID are needed by Google Analytics, which runs just after this script tag
 @InlineJs(ophanConfig().body, "ophanConfig.js")
 
+@if(WebpackSwitch.isSwitchedOn) {
```

Comment:
If webpack isn't switched on, what happens because you removed this all from `main.js`?